File: app/src/main/java/net/radityalabs/contactapp/presentation/util/RecycleViewUtil.java
package net.radityalabs.contactapp.presentation.util;

import android.content.Context;
import android.support.v7.widget.LinearLayoutManager;

/**
 * Created by radityagumay on 4/13/17.
 */

public class RecycleViewUtil {

    public static LinearLayoutManager linearLayoutManager(Context context) {
        LinearLayoutManager linearLayoutManager = new LinearLayoutManager(context);
        linearLayoutManager.setReverseLayout(false);
        linearLayoutManager.setStackFromEnd(true);
        return linearLayoutManager;
    }

    public static LinearLayoutManager simpleLinearLayoutManager(Context context) {
        LinearLayoutManager linearLayoutManager = new LinearLayoutManager(context);
        return linearLayoutManager;
    }
}

File: app/src/main/java/net/radityalabs/contactapp/presentation/util/RecycleViewUtil.kt
package net.radityalabs.contactapp.presentation.util

import android.content.Context
import android.support.v7.widget.LinearLayoutManager

/**
 * Created by radityagumay on 4/13/17.
 */

object RecycleViewUtil {

    fun linearLayoutManager(context: Context): LinearLayoutManager {
        val linearLayoutManager = LinearLayoutManager(context)
        linearLayoutManager.reverseLayout = false
        linearLayoutManager.stackFromEnd = true
        return linearLayoutManager
    }

    fun simpleLinearLayoutManager(context: Context): LinearLayoutManager {
        val linearLayoutManager = LinearLayoutManager(context)
        return linearLayoutManager
    }
}
